roles/configure/templates/bastion-cloud-config.yml.j2

```diff
@@ -52,6 +52,13 @@ write_files:
     wget https://storage.googleapis.com/kubernetes-release/release/v1.2.2/bin/linux/amd64/kubectl -O /tmp/kubectl
     chmod a+x /tmp/kubectl
     sudo mv /tmp/kubectl /usr/local/bin/kubectl
-
+    output={{ version.results[0].stdout }}
```

Comment:
Use more descriptive names here, please. s/output/version/ and s/size/version_length/